ui/src/cohortContext.ts

```diff
@@ -62,6 +68,14 @@ export function useNewCohortContext() {
     }
     setState(newState);
     await source.updateCohort(studyId, newState.present);
+
+    setState(
+      produce(newState, (state) => {
+        state.lastSaved = new Date();
```

Comment:
The backend also stores a `lastModified` property of the cohort. I don't think it makes much difference whether you use that one or this UI-generated one for the save state display. Noting in case it's easier to work with.

Reply:
No, good call. That's better since it reflects when changes were actually made.